Python/thread_pthread.h

```diff
@@ -302,6 +308,26 @@ PyThread_get_thread_ident(void)
     return (unsigned long) threadid;
 }
 
+#ifdef PY_HAVE_THREAD_NATIVE_ID
+unsigned long
+PyThread_get_thread_native_id(void)
+{
+    if (!initialized)
+        PyThread_init_thread();
+#ifdef __APPLE__
+    uint64_t native_id;
+    pthread_threadid_np(NULL, &native_id);
+#elif defined(__linux__)
+    pid_t native_id;
+    native_id = syscall(__NR_gettid);
```

Comment:
SYS_gettid exists as well. In posixmodule.c, we use "syscall(SYS_getrandom, ...)". In _posixsubprocess.c, we use "syscall(SYS_getdents64, ..."). I suggest to use SYS_gettid as well here.

To be clear, SYS_gettid and __NR_gettid are the same :-) https://stackoverflow.com/questions/22351033/difference-between-nr-gettid-and-sys-gettid I just prefer consistency in our code base ;-)